jarviscli/plugins/clear.py

```diff
@@ -1,8 +1,15 @@
-import os
+"""
+Jarvis plugin for clearing the terminal
+"""
 from plugin import plugin
+import os
 
 
-@plugin('clear')
+@plugin("clear")
 def clear(jarvis, s):
-    """Clears terminal"""
-    os.system("clear")
+    if os.name == "posix":
+        # Unix/Linux/MacOS/BSD/etc
+        os.system('clear')
+    elif os.name in ("nt", "dos", "ce"):
```

Comment:
Jarvis does the check for OS very early on, you can also use that instead of re-evaluating here.